### ISSUE TITLE
PeriodO questions

### ISSUE THREAD

**Issue by zoometh** [achp-project/cultural-heritage#7]
What to do with:

* `locator`: https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L14
* `spatialCoverageDescription` : https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L26
* `label`: https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L38
* `note` and `editorialNote`: https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L47-L48
* `source - locator`: https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L61
* `editorialNote` :https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L63

Is is better to point to a DOI rather than the DB url?:
* `url`: https://gist.github.com/rybesh/9f64c127ad8eeb69619896f22064bb0e#file-example-dataset-json-L54

What is the wikidata id of geographical regions such as:
* `Levant/Mesopotamia/Arabia`: https://github.com/achp-project/cultural-heritage/blob/05fdec5eca38b12c2a3945cb96ee6b5cbcea9f25/periodo-projects/exports/eamena_palaeolithic_levant_mesopotamia_arabia.json#L28

**Comment by zoometh** [comment]
- I changed the `locator` and `editorialNote` adding EAMENA's UUID of the period: https://github.com/achp-project/cultural-heritage/blob/main/periodo-projects/exports/eamena_palaeolithic_levant_mesopotamia_arabia.json#L59-L61
- I removed the wikidata entity when the period is composite: https://github.com/achp-project/cultural-heritage/blob/6d816b1207a03f7af49cc7332ff321bbd77f8a53/periodo-projects/exports/eamena_palaeolithic_levant_mesopotamia_arabia.json#L27
What do you think?

**Comment by yourvick** [comment]
I think it would be a real shame to lose the spatial information for all those periods - as around half of periods would be composite? Better to find a way of adding multiple entries to the json I reckon?

**Comment by yourvick** [comment]
What if we edited the json template to have the maximum number of nested spatialcoverages etc possible `label1`, `id1`, `label2`, `id1` etc etc and then just regexed the final json file to remove the numbers and any blank `spatialCoverage` records?

**Comment by zoometh** [comment]
Yes. For example: Mesopotamia [here](https://client.perio.do/?page=period-view&backendID=web-https%3A%2F%2Fdata.perio.do%2F&authorityID=p03wskd&periodID=p03wskdrkh7) with this spatial extent: https://github.com/achp-project/cultural-heritage/blob/1aeb29659e22ae4ce716fc64fdcde7c1d21522b8/periodo-projects/temp.json#L93-L110

**Comment by zoometh** [comment]
I have noticed, and corrected these errors in the [EAMENA period labels](https://github.com/achp-project/cultural-heritage/blob/main/periodo-projects/cultural_periods.tsv). To be compliant with EAMENA overall layout, cultures (eg. Ghaznavid, Timurid, Mamluk, etc.) should not be in parenthesis. Only the geographical region has to be in parenthesis. I've changed the parenthesis to brackets for:

- Dynastic Egypt _renamed_ Dynastic Egypt (Egypt)
- Islamic, Middle (Ghaznavid/Seljuq/Khwarazmshah) (Iran) _renamed_ Islamic, Middle [Ghaznavid/Seljuq/Khwarazmshah] (Iran)
- Islamic, Late (Timurid/Safavid/Qajar) (Iran) _renamed_ Islamic, Late [Timurid/Safavid/Qajar]
- Islamic, Early (Tahirid/Saffarid/Samanid/Buyids) (Iran)  _renamed_  Islamic, Early [Tahirid/Saffarid/Samanid/Buyids] (Iran) 
- Islamic, Middle (Mongol/Ilkhanid/Muzaffarid/Jalayrid) _renamed_ ...
- Islamic, Early (Ummayad/Abbasid) ...
- Islamic, Late (Mamluk/Rasulid) (Levant/Arabia) ...
- Islamic, Late (Ottoman) (Levant/Arabia)
- Islamic, Middle (Fatimid/Ayyubid/Crusader) (Levant/Arabia)
- Islamic, Early (Umayyad/Abbasid/Fatimid) (Levant/Arabia)
- Islamic, Early (Umayyad/Abbasid) (North Africa)
- Islamic, Middle (Fatimid/Zirid/Hammadid/Almoravid/Almohad) (North Africa)
- Islamic, Late (Ottoman/Saadi/Wattasid/Alaouite/Colonial) (North Africa)
- Islamic, Late (Ayyubid/Hafsid/Marinid/Zayyanid/Mamluk) (North Africa)
- Islamic, Late (Ottoman) (Iraq)
- Islamic, Early (Early Caliphate/Umayyad)
- Islamic, Middle (Abbasid)
- Islamic, Late (Post-Abbasid)

**Comment by zoometh** [comment]
In the same manner, these labels need to be changes in [rdm-bu-period.check.xlsx](https://github.com/achp-project/cultural-heritage/blob/main/periodo-projects/rdm-bu-period-check.xlsx). The results is:

- Islamic, Early [Umayyad/Abbasid/Fatimid] (Levant/Arabia)
- Islamic, Middle [Fatimid/Ayyubid] (Levant/Arabia)
- Islamic, Late [Mamluk/Rasulid] (Levant/Arabia)
- Islamic, Late [Ottoman] (Levant/Arabia)
- Islamic, Early [Early Caliphate/Umayyad] (Iraq)
- Islamic, Middle [Abbasid] (Iraq)
- Islamic, Late [Post-Abbasid] (Iraq)
- Islamic, Late [Ottoman) (Iraq)
- Islamic, Early [Ummayad/Abbasid] (Iran)
- Islamic, Early [Tahirid/Saffarid/Samanid/Buyids] (Iran)
- Islamic, Middle [Ghaznavid/Seljuq/Khwarazmshah] (Iran)
- Islamic, Middle [Mongol/Ilkhanid/Muzaffarid/Jalayrid] (Iran)
- Islamic, Late [Timurid/Safavid/Qajar] (Iran)
- Islamic, Early [Umayyad/Abbasid] (North Africa)
- Islamic, Middle [Fatimid/Zirid/Hammadid/Almoravid/Almohad] (North Africa)
- Islamic, Late [Ayyubid/Hafsid/Marinid/Zayyanid/Mamluk] (North Africa)
- Islamic, Late [Ottoman/Saadi/Wattasid/Alaouite/Colonial] (North Africa)
- ...
